src/mumble/ConnectDialog.cpp

```diff
@@ -470,7 +470,7 @@ QVariant ServerItem::data(int column, int role) const {
 				case 1:
 					return (dPing > 0.0) ? QString::number(uiPing) : QVariant();
 				case 2:
-					return uiUsers ? QString::fromLatin1("%1/%2 ").arg(uiUsers).arg(uiMaxUsers) : QVariant();
```

Comment:
This does not seem like the proper fix. From a quick glance `uiUsers` gets set if we actually have the necessary data. And as long as that is not the case, there's no point in showing user counts :eyes: